### PR TITLE
add project build instructions to README.md

### PR DESCRIPTION
**Pull request by xalopp**
@mmoll could you review this change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,67 @@
 # ExtJsPackages <a href="https://travis-ci.org/Mayflower/ExtJsPackages/"><img src="https://api.travis-ci.org/mayflower/ExtJsPackages.svg" alt="Build Status" style="max-width:100%;"></a>
 ExtJS Packages by Mayflower
 
+## Requirements
+
+* ExtJS-5.1 [direct download link](cdn.sencha.com/ext/gpl/ext-5.1.0-gpl.zip)
```
Comment (mmoll) [inline README.md]: missing `http://`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+* [Sencha Cmd](http://www.sencha.com/products/sencha-cmd/)
+* [JSDuck](https://github.com/senchalabs/jsduck)
+* [NodeJS](https://nodejs.org/)
+* [npm](https://www.npmjs.com/)
+
+## Setup & Build
+
+1. Install required node packages with npm:
```
Comment (mmoll) [inline README.md]: It should be stated in which directory all this should happen.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+    $ cd ExtJsPackages
+```
+* Checkout out a new branch to make your changes on:
+
+```sh
+    $ git checkout -b <your_new_patch>
+```
+* Have a look at [Sencha's Guide to JavaScript Style and Best Practices](https://github.com/sencha/code-guidelines)
+* Read the [JSDuck documentation](https://github.com/senchalabs/jsduck/wiki) and take a look at [JSduck example.js](https://github.com/senchalabs/jsduck/blob/master/opt/example.js).
+
```
Comment (mmoll) [inline README.md:54]: I'm missing a hint to add tests at least for new functionality.

**Comment by mmoll** [comment]
BTW, maybe it would make sense to split a part of this into a CONTRIBUTING.md file: https://github.com/blog/1184-contributing-guidelines

**Comment by xalopp** [comment]
@mmoll, @ritters, @andythedandy  could you review this change again?

**Comment by mmoll** [comment]
:+1:

**Pull request merged**